tests/data_generation.py

```diff
@@ -617,6 +638,7 @@ def _log_exp_composition_data(
         connection_args_list,
         space_list,
         n_samples_list,
+        smoke_data=None,
```

Comment:
this because. there are tests in `hypersphere`  that wants to test log_exp composition in some edge cases. It's better that we don't have to write another another duplication test for this I thought.